internal/gcs_test.go

```diff
@@ -0,0 +1,252 @@
+package internal
+
+import (
+	"cloud.google.com/go/storage"
+	"context"
+	"fmt"
+	"github.com/spf13/viper"
+	"github.com/stretchr/testify/assert"
+	"google.golang.org/api/googleapi"
+	"google.golang.org/api/iterator"
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+func getGcsBackend() (*GCSBackend, error) {
+	config, _ := getGcsConfig(viper.GetString("goofys.gcs.bucketWithPermission"))
+	gcsBackend, err := NewGCS(config)
+
+	return gcsBackend, err
+}
+
+var gcsBackend *GCSBackend;
+
+func TestGCS_BlobExist(t *testing.T) {
```

Comment:
what are you testing in the file? Whether you are using GCS apis correctly or not?

These tests rely on existnace of a gcs bucket with a particular set of files. To checkin this we need to find a way so that anyone without credentials can run the test.